packages/fxa-auth-server/lib/senders/templates/verify_login.html

```diff
@@ -1,86 +0,0 @@
-<!DOCTYPE html PUBLIC "-//W3C//DTD XHTML 1.0 Transitional//EN" "http://www.w3.org/TR/xhtml1/DTD/xhtml1-transitional.dtd">
-<html xmlns="http://www.w3.org/1999/xhtml">
-<head>
-  <meta http-equiv="Content-Type" content="text/html; charset=UTF-8" />
-  <title>{{t "Firefox Accounts"}}</title>
-</head>
-
-<body style="-ms-text-size-adjust: 100%; -webkit-text-size-adjust: 100%; margin: 0; padding: 0;">
-<table align="center" border="0" cellpadding="0" cellspacing="0" width="310" style="-webkit-text-size-adjust: 100%; border-collapse: collapse; mso-table-lspace: 0pt; mso-table-rspace: 0pt; width: 310px; margin: 0 auto;">
-
-<!--Logo-->
-<tr style="page-break-before: always">
-  <td align="center" id="firefox-logo" style="padding: 20px 0;">
-    {{^if sync}}
-      <img src="https://image.e.mozilla.org/lib/fe9915707361037e75/m/4/11c1e411-7dfe-4e04-914c-0f098edac96c.png" height="88" width="85" alt="" style="-ms-interpolation-mode: bicubic;" />
-    {{/if}}
-
-    {{#if sync}}
-      <img src="https://image.e.mozilla.org/lib/fe9915707361037e75/m/4/9f34e2a3-7947-4b5b-9c73-98faf8f49df6.png" height="137" width="270" alt="" style="-ms-interpolation-mode: bicubic;" />
-    {{/if}}
-  </td>
-</tr>
-
-
-<!--Header Area-->
-<tr style="page-break-before: always">
-  <td valign="top">
-    <h1 style="font-family: sans-serif; font-size: 21px; line-height: 29px; font-weight: normal; margin: 0 0 11px 0; text-align: center;">{{t "New sign-in to %(clientName)s" }}</h1>
-    <p class="primary" style="font-family: sans-serif; font-size: 14px; line-height: 21px; font-weight: normal; margin: 0 0 21px 0; text-align: center;">{{t "For added security, please confirm this sign-in from the following device:"}}<p/>
```

Comment:
There were a few examples like this too, where the author obviously intended to add a closing `</p>` but typoed it as `<p/>` by mistake, creating an extra, nested empty element instead of closing the parent one.